Services/CO.CDP.EntityVerification/appsettings.json

```diff
@@ -39,6 +39,6 @@
   },
   "Features": {
     "SwaggerUI": false,
-    "OutboxListener": false
+    "OutboxProcessorBackgroundService": true
```

Comment:
So, are we enabling the NOTIFY functionality on all environments at once, rather than rolling it out gradually?